lib/client-api/src/client_api.ts

```diff
@@ -123,6 +126,10 @@ export default class ClientApi {
       this._globalParameters.options
     );
 
+  getDisplayName = () =>
```

Comment:
This returns a function?

I think this method name is misleading

Reply:
`makeGetDisplayName`